README.rst

```diff
@@ -20,22 +31,28 @@ Example
     # Read the EDR file
     path = 'ener.edr'
     df = panedr.edr_to_df(path)
+    dic = panedr.edr_to_dict(path)
 
     # The `verbose` optional parameter can be set to True to display the
     # progress on stderr
     df = panedr.edr_to_df(path, verbose=True)
+    dic = panedr.edr_to_dict(path, verbose=True)
 
     # Get the average pressure after the first 10 ns
-    pressure_avg = df[u'Pressure'][df[u'Time'] > 10000].mean()
+    pressure_avg = df['Pressure'][df['Time'] > 10000].mean()
+    pressure_avg = dic['Pressure'][dic['Time'] > 10000].mean()
+
 
 Install
 -------
-
-Install the package with ``pip``:
+All code is found in panedrlite. Installing panedr installs panedrlite and pandas
```

Comment:
I would write the user facing docs with emphasis on *panedr* (as before) and not mention both together. This might be confusing. Anyone who isn't a developer will most likely just install panedr.

Then add a separate section on *panedrlite*  for developers.

Reply:
We will also need a conda package for panedrlite — presumably that's what @IAlibay has taken on?

Reply:
Yeah we'll deal with that once we set up deployment.

Reply:
Documentation is changed now